docs/recipes/Redux.md

```diff
@@ -0,0 +1,349 @@
+
+```terminal
+npx ignite-cli new ReduxApp --yes --removeDemo
+```
+
+If you are migrating an existing project these steps still apply, but you may need to migrate your existing state tree and other additional functionality.
+
+## Remove Mobx-State-Tree
+
+- Remove all Mobx-related dependencies from `package.json`, then run `yarn` or `npm i`
+
+```diff
+--"mobx": "6.10.2",
+--"mobx-react-lite": "4.0.5",
+--"mobx-state-tree": "5.3.0",
+
+--"reactotron-mst": "3.1.5",
+```
+
+- Ignite created default boilerplate Mobx-State-Tree files in the `models/` directory. Remove this entire directory and all files within it, these are not needed for Redux.
+
+- In `devtools/ReactotronConfig.ts` remove the `reactotron-mst` plugin. We can come back to [add a Redux plugin](#reactotron-support) later.
+
+```diff
+--import { mst } from "reactotron-mst"
+
+...
+
+const reactotron = Reactotron.configure({
+  name: require("../../package.json").name,
+  onConnect: () => {
+    /** since this file gets hot reloaded, let's clear the past logs every time we connect */
+    Reactotron.clear()
+  },
+--}).use(
+--  mst({
+--    /** ignore some chatty `mobx-state-tree` actions  */
+--    filter: (event) => /postProcessSnapshot|@APPLY_SNAPSHOT/.test(event.name) === false,
+--  }),
+--)
+++})
+```
+
+- Remove all `observer()` components and reformat as normal React components. Do a project-wide search for `observer(` and replace each component instance with the following pattern:
+
+```diff
+--import { observer } from "mobx-react-lite"
+
+--export const WelcomeScreen: FC<WelcomeScreenProps> = observer(function WelcomeScreen(props) {
+++export const WelcomeScreen: FC<WelcomeScreenProps> = (props) => {
+    ...
+--})
+++}
+```
+
+- Remove old Mobx-State-Tree store initialization / hydration code in `app.tsx`.
+- Call `hideSplashScreen` in a `useEffect` so the app loads for now. We'll replace this code when we add [persistence](#persistence) below.
+
+```diff
+--import { useInitialRootStore } from "./models"
+
+--const { rehydrated } = useInitialRootStore(() => {
+--setTimeout(hideSplashScreen, 500)
+--})
+++useEffect(() => {
+++    setTimeout(hideSplashScreen, 500)
+++}, [])
+
+--if (!rehydrated || !isNavigationStateRestored || !areFontsLoaded) return null
+++if (!isNavigationStateRestored || !areFontsLoaded) return null
+```
+
+You should be able to build and run your app! It won't have any data...but it's a good idea to check that it successfully runs before we move on.
+
+## Add Redux
+
+#### Install dependencies
+
+[redux-tooklit is the current recommended approach](https://redux.js.org/introduction/getting-started#redux-toolkit), and you'll also need `react-redux` bindings for your React Native app.
+
+```bash
+yarn add @reduxjs/toolkit
+yarn add react-redux
+```
+
+#### Create Store
+
+- In a new file `store.ts`, create your Redux store.
```

Comment:
what's your advice for the entire path?

```
./app/redux/store.ts
```
or
```
./app/store.ts
```

Reply:
I'm not strongly opinionated, IMO it doesn't matter too much, but it seems like Redux docs suggest a `store.ts` at more top level `app/store.ts` and then creating individual slices within feature folders. 
https://redux.js.org/style-guide/#structure-files-as-feature-folders-with-single-file-logic

I'll add the full file path to be consistent with Redux's suggestion 👍